pandas/tests/test_base.py

```diff
@@ -928,54 +927,51 @@ def test_validate_bool_args(self):
             with pytest.raises(ValueError):
                 self.int_series.drop_duplicates(inplace=value)
 
-    def test_getitem(self):
-        for i in self.indexes:
-            s = pd.Series(i)
+    def test_getitem(self, all_indices_fixture):
+        idx = all_indices_fixture
+        s = pd.Series(idx)
 
-            assert i[0] == s.iloc[0]
-            assert i[5] == s.iloc[5]
-            assert i[-1] == s.iloc[-1]
+        assert idx[0] == s.iloc[0]
+        assert idx[5] == s.iloc[5]
+        assert idx[-1] == s.iloc[-1]
 
-            assert i[-1] == i[9]
+        assert idx[-1] == idx[9]
 
-            with pytest.raises(IndexError):
-                i[20]
-            with pytest.raises(IndexError):
-                s.iloc[20]
+        pytest.raises(IndexError, idx.__getitem__, 20)
```

Comment:
it looks like recent changes on master not merged

Reply:
done